src/jobservice/job/impl/default_context.go

```diff
@@ -0,0 +1,151 @@
+// Copyright Project Harbor Authors. All rights reserved.
```

Comment:
Remove "All rights reserved" to match existing headers (see [this](https://github.com/goharbor/harbor/blob/master/src/common/config/config.go) as example).